### ISSUE TITLE
Allowing "unknown keys" to be used for item shortcuts

### ISSUE THREAD

**Issue by Vilabnotavailable** [CleverRaven/Cataclysm-DDA#55414]
### Is your feature request related to a problem? Please describe.

Hi, I have a French AZERTY keyboard and run the game in English on windows 10, and i can't use some "French specific key ?" to make item shortcuts.

Those keyboard keys are identified by the game as " unknown key 232" for exemple, and can be used to assign all the controls under the "keybinding menu".

However when the game ask me to enter a new letter to assign to items, it doesnt take into account some non standart keys, such as é, ', è, à, ù.... the prompt just doesn't register them and won't go away until i press ESC or a known key.

This behaviour notably prevent the user from assigning some of the key among the row which is right above the standart letters on the keyboard : **_²_** & **_é_** " **_'_** ( - **_è_** _ **_ç à_** ) =

This rows is usually the one which is used for items/action shortcuts on most game, having "holes" in that row is slightly annoying and this issues should be fixed.

### Solution you would like.

The prompt for assigning item shortcuts should take into account "unknown keys XX", similarly to how the keybinding menu works

**Comment by Night-Pryanik** [comment]
Closing as stale, since stalebot can't do it by itself.
